pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -65,6 +65,10 @@ def local_rank(self) -> int:
     def world_size(self) -> int:
         return self.num_processes
 
+    @property
+    def root_device(self) -> torch.device:
+        return self.device
```

Comment:
Is it correct, we aren't allowed to call xm.xla_device() here before we have spawned the processes?

Reply:
@awaelchli I don't think it would be required to call `root_device` outside the new processes.